### DISCUSSION TITLE
Laptop does not recognize Wifi

### DISCUSSION THREAD

**Discussion by henriquelucas** [LingmoOS/Community#8] [Bugs Report/缺陷提交] [answered]
I installed it on a Notebook where it has the 00:14.3 Network controller: Intel Corporation Comet Lake PCH-LP CNVi WiFi
However, the WiFi does not work, although in other versions such as Ubuntu it works correctly. Could you help me with this issue

**Comment by chun-awa** [comment, marked as answer]
use the latest image: https://packages.lingmo.org/lingmo/release/hydrogen-202409081130/iso/LingmoOS-2.0-Hydrogen-Release-desktop-amd64.iso

if the issue persists, try to install the [`firmware-iwlwifi`](https://packages.debian.org/sid/firmware-iwlwifi) package.
